### DISCUSSION TITLE
How to get back to full stock?

### DISCUSSION THREAD

**Discussion by adamhnat** [Boos4721/openwrt#34] [Q&A/答疑解惑] [answered]
Hi, How to back to fully stock firmware?
I already flashed stock image on mtd12, however then I found out that openwrt image flashed on mtd13 has different partition layout than original:
# cat /proc/mtd
dev:    size   erasesize  name
mtd0: 00100000 00020000 "0:SBL1"
mtd1: 00100000 00020000 "0:MIBIB"
mtd2: 00300000 00020000 "0:QSEE"
mtd3: 00080000 00020000 "0:DEVCFG"
mtd4: 00080000 00020000 "0:RPM"
mtd5: 00080000 00020000 "0:CDT"
mtd6: 00080000 00020000 "0:APPSBLENV"
mtd7: 00100000 00020000 "0:APPSBL"
mtd8: 00080000 00020000 "0:ART"
mtd9: 00080000 00020000 "bdata"
mtd10: 00080000 00020000 "crash"
mtd11: 00080000 00020000 "crash_syslog"
mtd12: 023c0000 00020000 "rootfs"
**mtd13: 08000000 00020000 "rootfs_1"**
mtd14: 01ec0000 00020000 "overlay"
mtd15: 00080000 00020000 "rsvd0"
mtd16: 00900000 00020000 "0:WIFIFW"
mtd17: 0041e000 0001f000 "kernel"
**mtd18: 01857000 0001f000 "ubi_rootfs"**
mtd19: 01876000 0001f000 "data"

How to get back to stock partition layout? is it possible? Vendor partition layout:
mtd13: 023c0000 00020000 "rootfs_1"
mtd17: 0160a000 0001f000 "ubi_rootfs"

**Comment by Boos4721** [comment]
**EZ, DOWNLOAD OFFICIAL MI FLASH TOOL FOR ROUTER,SELETC BOOT PARTITION TO `ROOTFS_0`, THEN FLASH UR ROUTER BY TOOL.**

**Comment by Boos4721** [comment, marked as answer]
- [FLASH TOOL LINK](http://bigota.miwifi.com/xiaoqiang/tools/MIWIFIRepairTool.x86.zip)
- [LASTEST FIRWARE ](https://cdn.cnbj1.fds.api.mi-img.com/xiaoqiang/rom/r3600/miwifi_r3600_firmware_aba17_1.1.19.bin)
